skt/runner.py

```diff
@@ -316,6 +327,14 @@ def __getresults(self, jobid=None):
             (ret, _) = self.__jobresult(jobid)
 
         if jobid is None or ret != 0:
+            if self.failures:
+                all_aborted = all([data[1] == ('Warn', 'Aborted')
+                                   for data in self.failures.values()])
```

Comment:
`all_aborted` will be true if `self.failures.values()` is empty. Would that be possible and would that be correct?

Reply:
Yes, that's exactly why there's the `if self.failures:` check before. So the check is skipped if there are no values and this quirk of `all()` doesn't bite us.

Reply:
I'm blind. I looked and looked for the check, and still didn't find it :grin:.